lib/openhab/core/items/item.rb

```diff
@@ -50,6 +50,22 @@ def groups
           group_names.map { |name| EntityLookup.lookup_item(name) }.compact
         end
 
+        #
+        # Checks if this item is a member of at least one of the given groups.
+        #
+        # @param groups [String, GroupItem] the group to check membership in
+        # @return [true, false]
+        #
+        # @example
+        #   event.item.member_of?(gFullOn)
+        #
+        def member_of?(*groups)
+          groups = groups.map! do |group|
```

Comment:
why do we have to set `groups = ` if we are also mutating groups directly with the bang?

Reply:
good catch, that's completely unnecessary. fixed in https://github.com/openhab/openhab-jruby/pull/9